packages/monitoring/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "url": "https://github.com/googleapis/googleapis",
```

Comment:
We can remove this line and instead put:

```json
  "repository": "googlecloudplatform/google-cloud-node"
```

so that users come to this repo.